src/app.ts

```diff
@@ -89,7 +89,7 @@ export const initializeAppModules = () => {
       initializedRequired: true,
       hasOutput: true,
       action: function (output) {
-        microsoftTeams.registerEnterSettingsHandler(function () {
+        microsoftTeams.registerChangeSettingsHandler(function () {

```

Comment:
If I change the name on line 88, it changes the element id in the DOM and I think that will break the tests unless they are also updated. Is my understanding correct? If so, I'll leave it unchanged to avoid having to update the tests.

Reply:
I don't see any tests on our side for this API. :| I think it's fine to update the name as well.